README.md

```diff
@@ -100,3 +100,40 @@ a template to create your ``.env`` file. For Bootcamp to work, it needs 6 values
 #### 8) Go to the home page of your bootcamp site and login via EdX
   - Log in as staff@example.com (password: edx).
   - You should see a message that you owe the amount of money equal to the installment you created in step 6.
+
+
+## Integration with FluidReview
+
+#### 1) Set FluidReview environment variables
+Values for the first four can be found at `../admin/developer/` on the FluidReview site.
```

Comment:
I think it would be clearer to list the URL as `<BOOTCAMP_ADMISSION_BASE_URL>/admin/developer/` similar to how it's formatted above. The `../` is throwing me off.

Reply:
✅